package.json

```diff
@@ -7,7 +7,8 @@
     "start": "expo start",
     "android": "expo start --android",
     "ios": "expo start --ios",
-    "eject": "expo eject"
+    "eject": "expo eject",
+    "server": "json-server -w server/db.json -r server/routes.json"
```

Comment:
I think we can remove this line, as we are not using `json-server` anymore, or at least expose `rooms.json`.